docs/governance/security-template.md

```diff
@@ -0,0 +1,142 @@
+* See the "**Alternative**" notes in the [OpenWallet Foundation's security vulnerability disclosure policy] document for how the "best practices" in this document can be changed to meet the needs of your project while still adhering to the OpenWallet Foundation's policies.
+
+Once your project's security vulnerability disclosure policy document is
+published, place a copy of it into each of your project's repositories, or
+create a `SECURITY.md` file in each of your project's repositories that links
+back to the primary document.
+
+[OpenWallet Foundation's security vulnerability disclosure policy]: https://tac.openwallet.foundation/governance/security/
+
+## About this Document
+This document document defines how security vulnerability reporting is handled
+in this project. The approach aligns with the [OpenWallet Foundation's security
+vulnerability disclosure policy]. Please review that document to understand
+the basis of the security reporting for this project
+
+This policy borrows heavily from the recommendations of the OpenSSF
+Vulnerability Disclosure working group. For up-to-date information on the latest
+recommendations related to vulnerability disclosures, please visit the [GitHub
+of that working group](https://github.com/ossf/wg-vulnerability-disclosures).
+
+If you are already familiar with what a security vulnerabulity disclosure policy
+is and are ready to report a vulnerability, please jump to [Report
+Intakes](#report-intakes).
+
+## What Is a Vulnerability Disclosure Policy?
+
+No piece of software is perfect. All software (at least, all software of a
+certain size and complexity) has bugs. In open source development, members of
+the community or the public find bugs and report them to the project. A
+vulnerability disclosure policy explains how this process functions from the
+perspective of the project.
+
+This vulnerability disclosure policy explains the rules and guidelines for
+this project. It is intended to act as both a reference for
+outsiders–including both bug reporters and those looking for information on the
+project’s security practices–as well as a set of rules that maintainers and
+contributors have agreed to follow.
+
+## Report Intakes
+
+This project has the following ways to submit security
+vulnerabilities. While the security team members will do their best to
+respond to bugs disclosed in all possible ways, it is encouraged for bug
+finders to report through the following approved channels:
+
+- Email the [OpenWallet Foundation's security list]: To report a security issue, please send an email with the name of the project/repository, a description of the issue, the steps you took to create the issue, affected versions, and if known, mitigations. If in triaging the email, the security team determines the issue may be a security vulnerability, a [GitHub security vulnerability report] will be opened.
```

Comment:
Emails can be intercepted by unauthorized individuals. Should we encourage the use of a secure channel, such as PGP-encrypted email bodies?

In addition, it is important to consider the security of online systems that use the codebase in question in production. A leaked and unresolved vulnerability could put these systems at risk.

Does the Linux Foundation have a public key available for such a purpose?

Reply:
@ryjones : Do we have such a system?

Reply:
GitHub supports "[Privately reporting and security vulnerability](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing-information-about-vulnerabilities/privately-reporting-a-security-vulnerability)." Until we have such a secure infrastructure in place, I would propose suggesting this as our preferred (if not only) option.

Reply:
Removed email list.